packages/http-client/src/index.ts

```diff
@@ -275,11 +275,11 @@ export class HttpClient {
       Headers.Accept,
       MediaTypes.ApplicationJson
     )
-    additionalHeaders[Headers.ContentType] = this._getExistingOrDefaultHeader(
-      additionalHeaders,
-      Headers.ContentType,
-      MediaTypes.ApplicationJson
-    )
+    additionalHeaders[Headers.ContentType] =
```

Comment:
What is the motivation for this change?

Reply:
For these errors: https://github.com/actions/toolkit/actions/runs/16661667415/job/47159994489